packages/pn-personafisica-webapp/src/pages/NotificationDetail.page.tsx

```diff
@@ -443,42 +400,45 @@ const NotificationDetail = () => {
   );
 
   const trackEventPaymentRecipient = (event: EventPaymentRecipientType, param?: object) => {
-    // eslint-disable-next-line functional/no-let
-    trackEventByType(
-      event as unknown as TrackEventType,
-      event === EventPaymentRecipientType.SEND_PAYMENT_STATUS ? param : undefined
+    PFEventStrategyFactory.triggerEvent(
+      event as unknown as PFEventsType,
```

Comment:
Avoid the use of unknown. In this case it should be useless

Reply:
added TODO on `EventPaymentRecipientType`